editor/src/components/canvas/__snapshots__/dom-walker.spec.browser.tsx.snap

```diff
@@ -3,35 +3,6 @@
 exports[`DOM Walker tests Label carried through for generated elements 1`] = `
 Object {
   "components": Array [
-    Object {
```

Comment:
I assume this is fine because no one relies on the fake storyboard scene anymore?

Reply:
Yep! feelsgoodman.gif